clients/scanoss/src/main/kotlin/model/Model.kt

```diff
@@ -67,6 +67,17 @@ data class License(
     val osadlUpdated: String? = null
 )
 
+/**
```

Comment:
Commit message nits:
- "of *the* scanoss-junit-4.12.json` test file"
- "by *the* SCANOSS team"
- "in a new commit" is clear and could be dropped IMO